plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -569,6 +569,17 @@ protected function handle_row_actions( $item, $column_name, $primary ) : string
 			);
 		}
 
+		/**
+		 * Filters the action links displayed for each review in the Reviews list table.
+		 *
+		 * @param string[]   $actions An array of comment actions. Default actions include:
+		 *                            'Approve', 'Unapprove', 'Edit', 'Reply', 'Spam',
+		 *                            'Delete', and 'Trash'.
+		 * @param WP_Comment $item The comment object.
+	 	*/
+		
+		$actions = apply_filters( 'comment_row_actions', array_filter( $actions ), $item );
+
```

Comment:
Lint error: `Whitespace found at end of linePHP_CodeSniffer(Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)`

```suggestion
```